source/view/blocks.js

```diff
@@ -4867,8 +4852,8 @@ Blockly.Blocks['init_nano_construction'] = {
   init: function() {
     this.appendDummyInput()
         .appendField("Start Construction");
-    this.setPreviousStatement(true);
-    this.setNextStatement(true);
+    this.setPreviousStatement(true,'Normal');
+    this.setNextStatement(true,'Normal');
```

Comment:
It makes it so these blocks can only connect to blocks with connection statements set to nothing (this.setPreviousStatement(true)) or blocks that have the same string (can also be an array of types)... so a this.setNextStatement(true,'Normal') will only allow blocks with their setPreviousStatement set to 'Normal' or nothing.